cmd/tracee-rules/main.go

```diff
@@ -27,6 +29,8 @@ const (
 	signatureBufferFlag = "sig-buffer"
 )
 
+var inputs engine.EventSources
```

Comment:
What's the purpose of moving this?

Reply:
feeding the signatures as events back to the input channel.